### PR TITLE
fix(APP_widget/ver.md): fix wrong desc and add details

### PR DESCRIPTION
**Pull request by cxw620**
RE from `iBiliPlayer-apinkRelease-8.0.0-b15088473`.

To be honest, managing API docs with md is really something weird, for example each contributor may explain the same thing with different ways of expression...

### CODE REVIEW

File: docs/APP_widget/ver.md
Comment (cxw620) [reply docs/APP_widget/ver.md:37]: image
64bit, 8.11.0

Without buvid:
image
64bit, 8.1.0

Without app-key:
image
32bit, 8.11.0

Comment (SessionHu) [reply docs/APP_widget/ver.md:37]: You are right. It is my problem, I have not noticed `buvid` before.
But in before documents, if a missing param or header do not lead to invalid response, it can be marked as `optional`. I think you can add these things to `Note` instead of simply `required`.

Comment (cxw620) [reply docs/APP_widget/ver.md:37]: > You are right. It is my problem, I have not noticed `buvid` before. But in before documents, if a missing param or header do not lead to invalid response, it can be marked as `optional`. I think you can add these things to `Note` instead of simply `required`.

In case of that missing any of the two headers causes invalid response (wrong ABI or version out of date), marking as `require` makes sense.

Let @SocialSisterYi or any collaborator who has write access make the final decision.

**Review by SocialSisterYi — APPROVED** [review]

**Pull request merged**
